smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -109,6 +110,9 @@ class Program
         [Option("--verify-data-from-module", Description = "Verify if a given module sent data do IoTHub.")]
         public string VerifyDataFromModule { get; } = "tempSensor";
 
+        [Option("--runtime-log-level", Description = "Change Runtime log level for modules.")]
+        public string RuntimeLogLevel { get; } = "debug";
```

Comment:
is it better to use enum instead of string type?

Reply:
Cool! Yes! Changed.